airgun/entities/host_new.py

```diff
@@ -311,6 +311,18 @@ def apply_erratas(self, entity_name, search):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def get_errata_pagination(self, entity_name):
+        """Get pagination info from Errata tab on selected host"""
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
+        view.content.errata.select()
+        # pass empty search to show any and all,
+        # if a previous method left the searchbar filled
+        view.content.errata.searchbar.fill(' ')
```

Comment:
Please correct this. We won't be applying the `'  '` string

Reply:
Yes, we made a new method to `clear` the SearchInput searchbar (widgets.py), which was unintentionally left occupied by `search` value passed to `apply_erratas` method (host_new.py).
Now, we clear the searchbar after selecting and applying erratas by search, so that the passed `search` does not persist.